.editorconfig

```diff
@@ -0,0 +1,12 @@
+# Top-most EditorConfig file
+root = true
+
+[*]
+charset = utf-8
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.{kt,kts}]
+indent_style = tab
+indent_size = 4
+max_line_length = 120
```

Comment:
@M-G12 @samn91 
anybody knows how to enforce trailing comma? 
I found this for scala `ij_scala_trailing_comma_mode =trailing_comma_add_when_multiline` but nothing for kotlin
also it is strange that it is only for "ij" i think it is for intelliJ ?